test/parseTest.ts

```diff
@@ -1,12 +1,63 @@
 import { assert } from "chai";
-import { parse } from "../src/parse";
+import { Parser } from "../src/Parser";
+import { defaultSchema } from "../src/schema";
 import { FileInfo, filesIn } from "./utils";
 
-describe("parse()", () => {
+/* tslint:disable:only-arrow-functions */
+
+describe("parse()", function() {
 	for (const output of filesIn("test/output/parse", ".json")) {
-		it(`works with ${output.name}`, () => {
+		beforeEach(function() {
```

Comment:
This should be `before`, not `beforeEach`

Reply:
Fixed 😄